packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_controller.dart

```diff
@@ -139,18 +139,20 @@ class DisplayOptions {
 
 class DeepLinksController extends DisposableController {
```

Comment:
mix in AutoDisposeControllerMixin and then you can add these listeners with addAutoDisposeListener, which means you don't have to manually remove them in dispose

Reply:
nice! thanks!